DB/migration/pending/3.sql

```diff
@@ -0,0 +1,47 @@
+create sequence "public"."service_type_price_id_seq";
+
+create table "public"."service_type_price" (
+    "id" integer not null default nextval('service_type_price_id_seq'::regclass),
+    "supplier_code" integer not null,
+    "service_type_id" integer not null,
+    "region_id" integer not null,
+    "date_from" timestamp without time zone not null,
```

Comment:
if you only want date, no times, there's a column type for that https://www.postgresql.org/docs/current/static/datatype-datetime.html

Reply:
Yeah - uncertain with this whether we need time or not